build.rs

```diff
@@ -391,6 +412,10 @@ fn build_c_code(target: &Target, pregenerated: PathBuf, out_dir: &Path, ring_cor
         }
     }
 
+    if target.arch == "mips" && target.endian == "big" {
```

Comment:
No need for `target.arch == "mips"` because no big-endian targets are supported yet.